doc/output-details-and-examples/src/output-files/eplusout-dfs.tex

```diff
@@ -1,103 +1,66 @@
-The Window State column has three possible outputs -- Base Window indicating windows without shades; a numeric value, from 0.0 to 180.0 for every 10.0 step, indicating the slat angle (in degrees) of the blind shading the window; blink indicating windows in shaded state by shades, screens, and blinds with fixed slat angle.
+The Window State column has three possible outputs -- ``Base Window'' indicating windows without shades; a numeric value, from 0.0 to 180.0 for every 10.0 step, indicating the slat angle (in degrees) of the blind shading the window; ``Blind or Slat Applied'' indicating windows in shaded state by shades, screens, and blinds with fixed slat angle.
 
-The third line describes the hourly daylight factors. Note that the last 4 columns are 0.0 because there is only one daylight reference point.:
+The lines after it describes the hourly daylight factors at each reference point:
 
 {\scriptsize
-\begin{longtable}[c]{>{\raggedright}p{0.5in}>{\raggedright}p{0.6in}>{\raggedright}p{0.6in}>{\raggedright}p{0.6in}>{\raggedright}p{0.6in}>{\raggedright}p{0.6in}>{\raggedright}p{0.6in}>{\raggedright}p{0.6in}>{\raggedright}p{0.6in}}
-\toprule 
-Hour & Daylight factor for clear sky at reference point 1 & Daylight factor for clear turbid sky at reference point 1 & Daylight factor for intermediate sky at reference point 1 & Daylight factor for overcast sky at reference point 1 & Daylight factor for clear sky at reference point 2 & Daylight factor for clear turbid sky at reference point 2 & Daylight factor for intermediate sky at reference point 2 & Daylight factor for overcast sky at reference point 2 \tabularnewline
-\midrule
-\endfirsthead
-
-\toprule 
-Hour & Daylight factor for clear sky at reference point 1 & Daylight factor for clear turbid sky at reference point 1 & Daylight factor for intermediate sky at reference point 1 & Daylight factor for overcast sky at reference point 1 & Daylight factor for clear sky at reference point 2 & Daylight factor for clear turbid sky at reference point 2 & Daylight factor for intermediate sky at reference point 2 & Daylight factor for overcast sky at reference point 2 \tabularnewline
-\midrule
+\begin{longtable}[c]{p{0.2in}p{1.5in}p{1.0in}p{1.0in}p{1.0in}p{1.0in}}
+\caption{Hourly Daylight Factors for the considered window at each reference point and 4 CIE Sky Conditions}
+\label{tab:dfs-output-at-each-ref-pt}\\
+\hline
+Hour & Reference Point & Daylight Factor for Clear Sky & Daylight Factor for Clear Turbid Sky & Daylight Factor for Intermediate Sky & Daylight Factor for Overcast Sky \\ \hline
 \endhead
-
-10 & 0.06239 & 0.07088 & 0.03377 & 0.01427 & 0.0 & 0.0 & 0.0 & 0.0 \tabularnewline
-\bottomrule
+%
+9 & ZN\_1\_DAYLREFPT1 & 0.021466 & 0.017368 & 0.013824 & 0.014127 \\ \hline
+9 & ZN\_1\_DAYLREFPT2 & 0.021466 & 0.017368 & 0.013824 & 0.014127 \\ \hline
+9 & ZN\_1\_DAYLREFPT3 & 0.021465 & 0.017368 & 0.013824 & 0.014127 \\ \hline
+10 & ZN\_1\_DAYLREFPT1 & 0.020245 & 0.016029 & 0.012201 & 0.014127 \\ \hline
```

Comment:
Update the eplusout.dfs:

![image](https://user-images.githubusercontent.com/5479063/83018803-28af4d00-a026-11ea-9e54-fee34f473382.png)